rasa/cli/test.py

```diff
@@ -17,6 +17,8 @@
 )
 import rasa.shared.utils.validation as validation_utils
 import rasa.cli.utils
+from rasa.shared.utils.cli import print_info
```

Comment:
```suggestion
import rasa.shared.utils.cli
```

We don't import methods directly. Instead we import the module they came from and call them explicitly.

Reply:
And move to line 9, after the other `import rasa` statements :)